examples/pq-aes.js

```diff
@@ -0,0 +1,119 @@
+let dataparty_crypto = require('../dist')
+
+
+const tweetnacl = require('tweetnacl')
+const hkdf = require('@panva/hkdf')
+const base64 = require('@stablelib/base64')
+
+const { siv } = require('@noble/ciphers/aes')
+
+class FullIdentity {
+    constructor(){
+        this.naclIdentity = new dataparty_crypto.Identity()
+        this.pqIdentity = new dataparty_crypto.PQIdentity()
+
+        this.naclSharedSecret = null
+        this.pqSharedSecret = null
+    }
+
+    async getStream(toIdentity){
+        this.naclSharedSecret = await dataparty_crypto.Routines.createNaclSharedSecret(
```

Comment:
Library implementation of stream helpers should not store secrets after the stream is constructed.